src/main/resources/META-INF/plugin.xml

```diff
@@ -115,7 +121,7 @@
 
             <action id="GitMachete.OpenMacheteFileAction"
                     class="com.virtuslab.gitmachete.frontend.actions.toolbar.OpenMacheteFileAction"
-                    icon="MacheteIcons.TEXT_FILE">
```

Comment:
Funnily enough, an typo in `icon` here causes an exception tho:

```
com.intellij.diagnostic.PluginException: Icon cannot be found in 'MacheteIcons.LOLXD', action 'class com.virtuslab.gitmachete.frontend.actions.toolbar.HelpAction'
```

(unlike apparently with a typo in `loadIcon` param)